ord_schema/interface/query.py

```diff
@@ -48,6 +199,47 @@ def __init__(self, dbname, user, password, host, port):
     def cursor(self):
         return self._connection.cursor()
 
+    def predicate_search(self, predicate):
```

Comment:
The naming here is a bit ambiguous; `predicate` is actually a set of predicates.